en_ewt-ud-test.conllu

```diff
@@ -15757,7 +15757,7 @@
 9	,	,	PUNCT	,	_	12	punct	12:punct	_
 10-11	Dinners	_	_	_	_	_	_	_	_
 10	Dinner	Dinner	PROPN	NNP	Number=Sing	12	nmod:poss	12:nmod:poss	_
-11	s	's	PART	POS	Typo=Yes	10	case	10:case	_
+11	s	's	PART	POS	Typo=Yes	10	case	10:case	CorrectForm='s
```

Comment:
Typo: should be [Diners Club](https://en.wikipedia.org/wiki/Diners_Club_International), no possessive